ddprofiler/app/src/main/java/ddprofiler/analysis/TextualAnalyzer.java

```diff
@@ -7,26 +7,26 @@
 import java.util.Iterator;
 import java.util.List;
 
-import ddprofiler.analysis.modules.Cardinality;
-import ddprofiler.analysis.modules.CardinalityAnalyzer;
-import ddprofiler.analysis.modules.Entities;
-import ddprofiler.analysis.modules.EntityAnalyzer;
-import ddprofiler.analysis.modules.KMinHash;
+import ddprofiler.analysis.modules.*;
+import xsystem.layers.XStructure;
 
 public class TextualAnalyzer implements TextualAnalysis {
 
     private List<DataConsumer> analyzers;
     private CardinalityAnalyzer ca;
     private KMinHash mh;
+    private XSystemAnalyzer xa;
```

Comment:
great.

can we create a new "config" in the project to choose what analyzers to "load"? That way we could activate/deactivate analyzers that, for example, consume too much resources (such as xsystem), if we have to.